modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -380,13 +401,16 @@ openshift-image-registry-installer-cloud-credentials-credentials.yaml
 openshift-ingress-operator-cloud-credentials-credentials.yaml
 openshift-machine-api-aws-cloud-credentials-credentials.yaml
 ----
-//Would love a GCP version of the above output.
+//Would love a GCP and Azure version of the above output.
```

Comment:
I couldn't find CI for GCP, but [searching Prow for `*azure*oidc*`][1] turned up [this 4.15 run][2] with:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-cloud-credential-operator-release-4.15-periodics-e2e-azure-manual-oidc/1708270693868638208/artifacts/e2e-azure-manual-oidc/ipi-conf-azure-oidc-creds-provision/build-log.txt | grep /tmp/manifests
2023/10/01 00:11:25 Wrote cluster authentication manifest at path /tmp/manifests/cluster-authentication-02-config.yaml
2023/10/01 00:11:25 Saved Azure AD pod identity webhook configuration to: /tmp/manifests/azure-ad-pod-identity-webhook-config.yaml
2023/10/01 00:12:27 Saved credentials configuration to: /tmp/manifests/openshift-cloud-controller-manager-azure-cloud-credentials-credentials.yaml
2023/10/01 00:13:05 Saved credentials configuration to: /tmp/manifests/openshift-machine-api-azure-cloud-credentials-credentials.yaml
2023/10/01 00:13:38 Saved credentials configuration to: /tmp/manifests/openshift-image-registry-installer-cloud-credentials-credentials.yaml
2023/10/01 00:14:04 Saved credentials configuration to: /tmp/manifests/openshift-ingress-operator-cloud-credentials-credentials.yaml
2023/10/01 00:14:20 Saved credentials configuration to: /tmp/manifests/openshift-cloud-network-config-controller-cloud-credentials-credentials.yaml
2023/10/01 00:14:59 Saved credentials configuration to: /tmp/manifests/openshift-cluster-csi-drivers-azure-disk-credentials-credentials.yaml
2023/10/01 00:15:38 Saved credentials configuration to: /tmp/manifests/openshift-cluster-csi-drivers-azure-file-credentials-credentials.yaml
```

[1]: https://prow.ci.openshift.org/?job=*azure*oidc*
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-cloud-credential-operator-release-4.15-periodics-e2e-azure-manual-oidc/1708270693868638208

Reply:
Oh yeah, I actually have this one from Jeremiah in the other Azure PR. Will duplicate it in here